src/main/java/com/faforever/server/entity/Game.java

```diff
@@ -118,7 +118,7 @@
   @JoinColumn(name = "host")
   private Player host;
 
-  @ManyToOne(optional = false, fetch = FetchType.LAZY)
+  @ManyToOne(optional = false, fetch = FetchType.EAGER)
```

Comment:
Is there a specific reason we need eager loading here? If so, skip the definition because eager is default. If not, keep it lazy.

Reply:
This prevents an exception from hibernate. If I understood this correctly it is the following: 
When I access the MapVersion and the Map through the Game instance passed to ´onGameEnded(...)´ the hibernate transaction that loaded this Game proxy is already closed, so when I try to access the MapVersion or the Map without them being loaded eagerly I get some kind of "transaction not active" exception.